contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -11,6 +11,7 @@ packages:
 - "autoconf"
 - "automake"
 - "binutils"
+- "bison"
```

Comment:
Add `bison` to https://github.com/bitcoin/bitcoin/blob/master/depends/README.md#for-linux-including-i386-arm-cross-compilation?